crates/mapdiffbot2/src/github_processor.rs

```diff
@@ -110,35 +111,114 @@ async fn process_pull(
 
     job_sender.lock().await.send(job).await?;
 
-    log::trace!("Job sent to queue");
+    log::debug!("Job sent to queue");
 
     Ok(())
 }
 
-async fn handle_pull_request(payload: String, job_sender: DataJobSender) -> Result<&'static str> {
+async fn handle_pull_request(
+    payload: String,
+    job_sender: DataJobSender,
+    pool: actix_web::web::Data<Option<mysql_async::Pool>>,
+) -> Result<&'static str> {
     let payload: PullRequestEventPayload = serde_json::from_str(&payload)?;
-    if payload.action != "opened" && payload.action != "synchronize" {
-        return Ok("PR not opened or updated");
-    }
 
-    log::trace!("Creating checkrun");
+    let pool = pool.get_ref();
 
-    let check_run = CheckRun::create(
-        &payload.repository.full_name(),
-        &payload.pull_request.head.sha,
-        payload.installation.id,
-        Some("MapDiffBot2"),
-    )
-    .await?;
+    match payload.action.as_str() {
+        "opened" | "synchronize" | "reopened" => {
```

Comment:
reopened was deliberately not allowed to prevent spamming the service by closing and opening a PR repeatedly, is this no longer a concern?

Reply:
I think this is left over from testing. Or atleast I hope it is.

Reply:
Still a concern, map rendering takes up alot of space